### ISSUE TITLE
pacman makedepends list missing brotli-devel requirement.

### ISSUE THREAD

**Issue by earnie** [Alexpux/MSYS2-packages#1367]
As I was building pacman with makepkg I needed to install missing requirement of brotli-devel.  I had installed the identified missing ones early in the process but was surprised by the missing library later.

```make
/usr/lib/gcc/x86_64-pc/-usrmsys//7.3.0lib//..gcc//..x86_64/-.pc.-/msys./.7.3.0//x86_64.-.pc/-.msys.//bin./.ld/:.. /cannot x86_64find- pc--lbrotlidec
msys//usrbin//libld/:gcc /cannotx86_64 -findpc --msyslbrotlidec
//7.3.0usr//.lib.//gcc./.x86_64/-../pc.-.msys//7.3.0x86_64/-.pc.-/msys./.bin//.ld.:/ .cannot. /find x86_64--lbrotlicommonpc
-msys/bin/ld: cannot find -lbrotlicommon
collect2: error: ld returned 1 exit status
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:478: testpkg.exe] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [Makefile:474: cleanupdelta.exe] Error 1
make[1]: *** [Makefile:980: all-recursive] Error 1
make: *** [Makefile:889: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...

$ pacman -S brotli-devel
```

**Comment by Alexpux** [comment]
Ok this is missing dependency for lubcurl-devel package. I will fix

**Comment by Alexpux** [comment]
Will be fixed in 7.61.0-2 : https://github.com/Alexpux/MSYS2-packages/commit/4a6004110dcf242855fa214310443278dfcfafed
